src/resultsdisassemblypage.cpp

```diff
@@ -199,3 +201,46 @@ void ResultsDisassemblyPage::setData(const Data::DisassemblyResult &data)
         m_objdump = QLatin1String("aarch64-linux-gnu-objdump");
     }
 }
+
+void ResultsDisassemblyPage::resetCallStack()
+{
+    m_callStack.clear();
+}
+
+void calcFunction(QString asmLineCall, QString* offset, QString* symName)
+{
+    QStringList sym = asmLineCall.trimmed().split(QLatin1Char('<'));
+    *offset = sym[0].trimmed();
+    *symName = sym[1].replace(QLatin1Char('>'), QLatin1String(""));
+}
+
+void ResultsDisassemblyPage::jumpToAsmCallee(QModelIndex index)
+{
+    QStandardItem* asmItem = model->takeItem(index.row(), 0);
+    QString opCodeCall = m_arch.startsWith(QLatin1String("arm")) ? QLatin1String("bl") : QLatin1String("callq");
+    QString opCodeReturn = m_arch.startsWith(QLatin1String("arm")) ? QLatin1String("ret") : QLatin1String("retq");
+    QString asmLine = asmItem->text();
+    if (asmLine.contains(opCodeCall)) {
+        QString offset, symName;
+        QString asmLineSimple = asmLine.simplified();
+
+        calcFunction(asmLineSimple.section(opCodeCall, 1), &offset, &symName);
+
+        QHash<Data::Symbol, Data::DisassemblyEntry>::iterator i = m_disasmResult.entries.begin();
+        while (i != m_disasmResult.entries.end()) {
+            QString relAddr = QString::number(i.key().relAddr, 16);
```

Comment:
instead of converting every address to a string, just convert the offset to a number in calcFunction once